.ci/Dockerfile

```diff
@@ -1,8 +1,3 @@
-FROM dsluiuc/honeybadgermpc-docker-base AS travis-image
-
-WORKDIR /usr/src/HoneyBadgerMPC
-COPY . /usr/src/HoneyBadgerMPC
-
-RUN pip install -e .["tests,docs"]
-RUN make -C apps/shuffle/cpp -j
+FROM dsluiuc/honeybadgermpc-test-image
```

Comment:
Is this what we want to call it?

Reply:
I named it this to prevent a clash with the existing docker image. I think this ties in pretty well with your concerns about two people updating this.